Prevent non-evdev devices from firing the eventcomm AutoDevProbe callback.

Index: xserver-xorg-input-synaptics/conf/50-synaptics.conf
===================================================================
--- xserver-xorg-input-synaptics.orig/conf/50-synaptics.conf	2011-07-05 11:54:33.911347672 +0300
+++ xserver-xorg-input-synaptics/conf/50-synaptics.conf	2011-07-05 11:54:53.411347674 +0300
@@ -14,5 +14,5 @@
 # This option is recommend on all Linux systems using evdev, but cannot be
 # enabled by default. See the following link for details:
 # http://who-t.blogspot.com/2010/11/how-to-ignore-configuration-errors.html
-#       MatchDevicePath "/dev/input/event*"
+      MatchDevicePath "/dev/input/event*"
 EndSection
